docs/pages/store/reading-and-writing.mdx

```diff
@@ -36,7 +36,7 @@ tables: {
 ```solidity
 // Setting a record
 uint256[] memory fooArray = new uint256[](1);
```

Comment:
should this likewise be 

```suggestion
uint256[1] memory fooArray;
```

Reply:
No, because `fooArray` is a dynamic array in the table definition